docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -127,7 +127,7 @@ following properties will be applied before passing the request on to its actual
 [cols="<m,<m,<2",options="header"]
 |===
 |Property Name|Default|Description
-|quarkus.http.cors.origins||The comma-separated list of origins allowed for CORS. The filter allows any origin if this is not
+|quarkus.http.cors.origins|*|The comma-separated list of origins allowed for CORS. The filter allows any origin if this is not
```

Comment:
See my note about the default value. Perhaps it should read: `The filter allows any origin if this is not set or set to '*'' ?